server/models/data/Student.js

```diff
@@ -7,7 +7,7 @@ class Student {
         this.firstName = studentData.first_name;
         this.lastName = studentData.last_name;
         this.email = studentData.email;
-        this.type = UserTypes.student;
```

Comment:
Please note that this change was due to the conflicting issue with the worker type attribute.